### DISCUSSION TITLE
Is there a setting to disable the SQL Console: Messages panel popping up?

### DISCUSSION THREAD

**Discussion by pamelafox** [mtxr/vscode-sqltools#1188] [General]
This panel pops up next to terminal when I start up a Dev Container with SQLTools installed:

<img width="842" alt="Screenshot 2023-07-17 at 2 52 03 PM" src="https://github.com/mtxr/vscode-sqltools/assets/297042/95178111-b96f-43f6-8806-22a562aeb204">

Is there a setting for disabling that? I looked through the settings and didn't see an obvious option. Thanks!

**Comment by gjsjohnmurray** [comment]
I suggest you right-click on the "SQL CONSOLE: MESSAGES" header and choose "Reset Location". This will turn it into a tab on the panel. Then if you want to hide it, right-click on it again and choose "Hide SQL Console".

**Reply by pamelafox** [reply]
Thanks! I was hoping to find a setting that I can put in a devcontainer.json file to auto-hide it. We are making many code samples that include the SQLTools extension, but it's a bit jarring right now when the dev container opens and there's the large SQLConsole section.

**Reply by gjsjohnmurray** [reply]
After you reset its location so it's no longer on the TERMINAL view container tab on Panel, I'd only expect focus to switch to it one its own SQL CONSOLE: MESSAGES tab if you run a query that raises an error. Is that what you observe?

**Reply by pamelafox** [reply]
Yes, that's what I observe. However, the issue is the initial experience when the dev container starts up. We will have many developers trying out these samples that may not necessarily know what SQLTools is, so they may be confused by the big SQL Console upon startup. I would love to include the SQLTools extension in the devcontainer.json as a helpful tool for those who want it, but I also want devs to not be overwhelmed when first opening the project. That's why I'm hoping for a setting that would hide it from the get go. Thanks!

**Reply by elesq** [reply]
I would second that request for this feature

**Comment by gjsjohnmurray** [comment]
Today's release includes a PR intended to improve this case.
